tests/PageObjects/GamePage.ts

```diff
@@ -9,4 +9,28 @@ export class GamePagePO {
     await this.page.waitForTimeout(1500);
     await this.page.keyboard.press('Enter');
   }
+
+  public async restartSong() {
+    await this.page.getByTestId('button-restart-song').click();
+  }
+
+  public get playersScoreElement() {
+    return this.page.getByTestId('players-score');
+  }
+
+  public get currentPlayersScore() {
+    return this.playersScoreElement.getAttribute('data-score');
+  }
+
+  public async waitForPlayersScoreToBeGreaterThan(expected: number) {
+    await expect(async () => {
+      const p1score = await this.currentPlayersScore;
+
+      expect(parseInt(p1score!, 10)).toBeGreaterThan(expected);
+    }).toPass({ timeout: 15_000 });
+  }
+
+  public async expectPlayersScoreToBe() {
```

Comment:
should also expect an argument `expected: number`, just as the method you fixed above

Reply:
Aah, right, sorry. Now should be ok